### PR TITLE
Setup Sri Lanka environment

### PR DESCRIPTION
**Pull request by prabhanshuguptagit**
**Story card:** [ch4446](https://app.clubhouse.io/simpledotorg/story/4446/add-hosts-files), [ch4265](https://app.clubhouse.io/simpledotorg/story/4265/configure-facility-sizes), [ch4267](https://app.clubhouse.io/simpledotorg/story/4267/configure-medication-frequency-labels), [ch4264](https://app.clubhouse.io/simpledotorg/story/4264/configure-sri-lanka-address-fields), [ch4266](https://app.clubhouse.io/simpledotorg/story/4266/configure-canonical-state-list), [ch4269](https://app.clubhouse.io/simpledotorg/story/4269/add-business-identifier-to-server), [ch4448](https://app.clubhouse.io/simpledotorg/story/4448/setup-semaphore-to-deploy-to-sri-lanka)

## Because

This adds the Sri Lanka hosts to cap config. This also adds the country specific config required to get simple-server functional for a new `DEFAULT_COUNTRY` (`LK`).

[Sri Lanka deployment playbook](https://docs.google.com/document/d/11mPG_8HNXPXiT6vA6pvNskGJdE8KVfqG/edit#)

### CODE REVIEW

File: config/locales/en_LK.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        village_or_colony: "Town/Area"
+        zone: "Zone (TBD)"
+        district: "District"
+        state: "Province *"
+        country: "Country *"
+        pin: "Postal Code"
+      drug:
+        frequency:
+          one_per_day: "Daily"
+          two_per_day: "BD (morning and night)"
```
Comment (harimohanraj89) [inline config/locales/en_LK.yml:27]: Is this the copy shown in the app? Can we check if this length causes display/layout issues?

File: app/models/patient_business_identifier.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@ class PatientBusinessIdentifier < ApplicationRecord
   enum identifier_type: {
     simple_bp_passport: "simple_bp_passport",
     bangladesh_national_id: "bangladesh_national_id",
+    sri_lanka_national_id: "sri_lanka_national_id",
```
Comment (harimohanraj89) [inline app/models/patient_business_identifier.rb:10]: Do we need to add this to specs?

**Review by harimohanraj89 — APPROVED** [review]
Suggestions - not critical blockers.

File: app/models/patient_business_identifier.rb (hunk 2 above)
Comment (harimohanraj89) [reply app/models/patient_business_identifier.rb:10]: Sweet.

**Comment by harimohanraj89** [comment]
I see we're trying to suss out the impact of the long medication strings. I'll let you decide how we should handle this once that discussion is complete 👍

File: config/locales/en_LK.yml (hunk 1 above)
Comment (prabhanshuguptagit) [reply config/locales/en_LK.yml:27]: This copy is not being shown anywhere on the app. We will have to keep it in mind when we pick up https://app.clubhouse.io/simpledotorg/story/2915/implement-drugfrequencyfactory-drugfrequencyprovider-for-providing-drug-frequencies-label-depending-on-the-country.

**Pull request merged**